data/tooltips.txt

```diff
@@ -368,6 +368,9 @@ tip "movement, full / no cargo:"
 tip "thrust:"
 	`Thrust produced by engines. Higher thrust increases a ship's acceleration and top speed. The more mass a ship has, the more thrust is needed to achieve the same acceleration.`
 
+tip "sail thrust:"
+	`Thrust produced by engines, dependant on direction. Sail-style thrusters function optimally when facing directly towards or away from the gravitational center of a system, but facing perpendicularly to the center produces nearly zero thrust.`
```

Comment:
Sails (in real life), tend to be most effective when perpendicular to the wind for sloops.  When with an aft wind (rear pushing forward) spinnakers are effective.  The least effective direction for sail power is a headwind across the bow.  To travel into a headwind a sailboat travels diagonally to the wind to perform windward tacks.  This forms a zig-zag pattern while moving up-wind.

If we generically apply this to space; the least effective direction is into the sun.  Circle around the sun for the most effective and moving away from the sun will get you thrust but the further away from the center you get the less thrust.

Reply:
I wish I knew the mechanics and reasons for this, but I can't personally imagine (and this is personal incredulity yes) how pointing perpendicular to the direction of wind movement would be the most effective or efficient direction of travel...

If there is a better way to do this, I could refactoring all this code. I think a cross product would simplify some of the math considerably, for one.

Reply:
The sails slightly angle to form an air foil which creates a lower pressure in front while the higher pressure wind pushes up the rear is my understanding.  Worth noting that a sail boat can angle its sails for optimal propulsion regardless of travel direction to move forward so they're fairly efficient except for the head wind.

I tried using terms you could google such as:
- sloop
- windward tack
- leeward tack
- spinnaker

There's a lot more to operating a sail boat but I didn't think it relevant to your PR.

Reply:
Relevant: https://space.stackexchange.com/questions/370/can-you-tack-against-the-sun-using-a-solar-sail

Reply:
We also have drag in space in ES as well as top speed in space.  So we could explain away the sailing game mechanic the same way we explain drag and top speed.  https://discord.com/channels/251118043411775489/688532441324847150/895606038563278871